Makefile

```diff
@@ -12,7 +12,7 @@ help: ## Display this help text
 all: build test
 
 build: ## Build docker image
-	@docker build --file Dockerfile-one-container --tag=${IMAGE}:${VERSION} .
+	@docker build --file Dockerfile --tag=${IMAGE}:${VERSION} .
```

Comment:
I guess `--file Dockerfile` could be removed?, in any case, a very tiny nit, so no blocking at all

Reply:
```suggestion
	@docker build --tag=${IMAGE}:${VERSION} .
```